File: PHP/Car.php
<?php
require_once('account.php');
class Car
{
    public $id;
    public $license;
    public $driver;
    public $passegers;

    public function __constructor($license, $driver){
        $this->license = $license;
        $this->driver = $driver;
    }

    public function PrintDataCar(){
        echo "License: " + $this->license + " Name: " + $this->driver->name + " Document: " + $this->driver->document; 
      }

}
?>

File: Java/Car.java
public class Car {
    private Integer id;
    private Account driver;
    private String license;
    protected Integer passengers;
    
    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }
    
    public void printDataCar(){
        System.out.println("License: " + license + " Driver: " + driver.name
        + " Passegers: " + passengers);
    }

    public Integer getPassegers()
    {   
        return this.passengers;
    }

    public void setPassegers(Integer passengers)
    {
        if(passengers == 4)
            this.passengers=passengers;
        else
            System.out.println("You must assign four passengers");
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    
}
